mscorlib/system/globalization/charunicodeinfo.cs

```diff
@@ -322,7 +338,7 @@ internal static bool IsWhiteSpace(char c)
             }
             return (((double*)s_pNumericValues)[pBytePtr[(ch & 0x000f)]]);
 #else
-            return (((double*)s_pNumericValues)[pBytePtr[(ch & 0x000f)]]);
+            return EndianSwap(((double*)s_pNumericValues)[pBytePtr[(ch & 0x000f)]]);
```

Comment:
Should the other calls to EndianSwap() from 41a6d3d6a5432b964b6f8439b6375ffad6f2e6cb be changed accordingly?